src/app/core/header/header.component.html

```diff
@@ -2,11 +2,11 @@
     <div class="logo">
         <angular-logo></angular-logo>
     </div>
-    <div class="profile user">
-        <img class="user" src="assets/img/ic-user.svg">
-        User login
+    <div class="profile user" *ngIf="isAuth">
+        <img src="assets/img/ic-user.svg">
+        {{userName}}
     </div>
-    <div class="profile" (click)="exit()">
+    <div class="profile" (click)="exit()" *ngIf="isAuth">
```

Comment:
I suggest optimizing this part by using ng-container with one subscription.
In that case, we reduce subscriptions from two to one :)

something like that
```
<ng-container *ngIf="isAuth">
  <div class="profile user"> ...
  <div class="profile" (click)="exit()"> ...
```